ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -1090,24 +1091,22 @@ + (ORKOrderedTask *)timedWalkTaskWithIdentifier:(NSString *)identifier
             
             ORKStepArrayAddStep(steps, step);
         }
-        
+
+        {
+            ORKTimedWalkStep *step = [[ORKTimedWalkStep alloc] initWithIdentifier:ORKTimedWalkTurnAroundStepIdentifier];
+            step.title = ORKLocalizedString(@"TIMED_WALK_INSTRUCTION_TURN", nil);
+            step.text = ORKLocalizedString(@"TIMED_WALK_INSTRUCTION_TEXT", nil);
+            step.spokenInstruction = step.title;
+            step.recorderConfigurations = recorderConfigurations;
+            step.distanceInMeters = 1;
+            step.shouldTintImages = YES;
+            step.image = [UIImage imageNamed:@"turnaround" inBundle:[NSBundle bundleForClass:[self class]] compatibleWithTraitCollection:nil];
+            step.stepDuration = timeLimit == 0 ? CGFLOAT_MAX : timeLimit;
```

Comment:
@vtourraine How about we provide a parameter, but if the developer doesn't specify, we use a default value?